### PR TITLE
TPRUN-3432 Ensure secure Spring version.

### PR DESCRIPTION
**Pull request by andreasmattes**

### CODE REVIEW

File: activemq-karaf/src/main/resources/features.xml [hunk 1]
```diff
@@ -55,12 +55,13 @@
         Don't forget to bind Karaf JMX server to 0.0.0.0 to have a functional webconsole:
         In etc/org.apache.karaf.management.cfg, both rmiRegistryHost and rmiServerHost should use 0.0.0.0
     -->
-    <feature name="activemq-web-console" version="${upstream.version}">
```
Comment (jgreffe) [inline activemq-karaf/src/main/resources/features.xml:58]: Beware of this feature removal, if customer installed it before patch, customer won't be able to uninstall anymore.

**Review by jgreffe — CHANGES_REQUESTED** [review]

File: activemq-karaf/src/main/resources/features.xml (hunk 1 above)
Comment (andreasmattes) [reply activemq-karaf/src/main/resources/features.xml:58]: so we should rather add an empty feature I think

Comment (andreasmattes) [reply activemq-karaf/src/main/resources/features.xml:58]: done

**Review by jgreffe — APPROVED** [review]
Let's hope customers won't use `activemq-web-console` ....

**Pull request merged**